clusterdock/models.py

```diff
@@ -44,10 +45,19 @@ class Cluster:
 
     Args:
         *nodes: One or more :py:obj:`clusterdock.models.Node` instances.
+        name (:obj:`str`, optional): Cluster name to use. Default: :py:func:`generate_cluster_name`
```

Comment:
Default should be like:
`Default: a randomly-generated cluster name` (we don't need to include the implementation details in the docstring)